controller/src/main/java/io/pravega/controller/store/stream/InMemoryStream.java

```diff
@@ -321,21 +304,18 @@ public void refresh() {
     }
 
     @Override
-    CompletableFuture<Void> createHistoryTable(Data<Integer> data) {
+    CompletableFuture<Void> createHistoryTableIfNotExists(Data<Integer> data) {
```

Comment:
Because in ZkStream implementation we call store.createIfNotExist while creating. 
So changed to have the same behaviour in InMemoryStream. 
The name change makes it obvious that we create iff it does not exist. any previous data is not overwritten